viewer/packages/pointclouds/src/PointCloudPickingHandler.ts

```diff
@@ -83,21 +92,37 @@ export class PointCloudPickingHandler {
       .map(x => {
         const pointCloudNode = determinePointCloudNode(x.object, visibleNodes);
         if (pointCloudNode === null) {
-          throw new Error(`Coulds not find PointCloudNode for intersected point`);
+          throw new Error(`Could not find PointCloudNode for intersected point`);
         }
 
         const pointCloudObject = pointCloudNode.getStylableObjectMetadata(x.objectId);
-        const [annotationId, asset] =
-          pointCloudObject !== undefined ? [pointCloudObject.annotationId, pointCloudObject.assetRef] : [0, undefined];
+
+        let annotationId: number = 0;
```

Comment:
```suggestion
        let annotationId: number = -1;
```
And then we can remove the line `annotationId = -1;` below

Reply:
In hindsight - maybe we should keep `annotationId = 0` when the volume is a DM volume? That would more neatly fit with the previous semantic of an "unannotated" point - which is probably what users want when they haven't implemented DM support yet.